src/Interop/Ubiquity.NET.Llvm.Interop/NativeMethods.cs

```diff
@@ -19,61 +19,6 @@ namespace Ubiquity.NET.Llvm.Interop
     /// <summary>Interop methods for the Ubiquity.NET LibLLVM library</summary>
     public static partial class NativeMethods
```

Comment:
Is there a reason not to delete this file? It looks empty...

Reply:
The static string in there is needed by the infrastructure that uses the dll name, but I can put that into another file to avoid the seemingly empty file.